File: 3198/3198.c
int findCitiesInEachState(struct TreeNode* root) {
    if (!root) return 0;
    int l = findCitiesInEachState(root->left);
    int r = findCitiesInEachState(root->right);
    return 1 + (l > r ? l : r);
}

File: 3198/3198.js
/**
 * @param {TreeNode} root
 * @return {number}
 */
var findCitiesInEachState = function(root) {
    if (!root) return 0;
    return 1 + Math.max(findCitiesInEachState(root.left), findCitiesInEachState(root.right));
};